media-libs/libaacs/Manifest

```diff
@@ -1,2 +1,7 @@
 DIST libaacs-0.10.0.tar.bz2 319890 BLAKE2B b7ea746e72028f718ee630d74360c52627360b503d739e884c213651597e719711b2dc7649ded4164ee35a56e3420de59e420c4ce8fa7b1dbebcd547f5221798 SHA512 4527749c3c78fd270012b3b36e916356a4aeb8f90169e781a00739f01f190121691042607209864e9b676ede9741f614c6003a7dfbb350dc020e1fa94cd6d66e
+DIST libaacs-0.11.0.tar.bz2 324061 BLAKE2B 8faffa270580f82b7fc6ae53922a9ec15087b754adebc84f1fd216c748dbcbe6723c14908efc9ef77b891bda149d10bf5020321906a36a3654dddbf6bdc00936 SHA512 167edbb6f26599e41d6084908039bf902f69f57da1f64f1491734157d1568ad0b32c8ea3064e0706e383af8dc6007eef65170b2b47222bf3d363e395e0b60388
 DIST libaacs-0.9.0.tar.bz2 316323 BLAKE2B 4fd1ecca0ba5106b1d0195884356a8a8d7163614ebc2fe09f85ed79fa1534b6376951ab90670b3a7d375129e76fae80e3c42f3e1971d0cbcabd6e4e8384bde1a SHA512 b9c658db22a0966486070d3b762509a7c0120469ab29d1e57c1b624cad309701de8b92455ec649db4d731be4a5c29c38fb4f99bb7617dbb4254270284a1903f6
+EBUILD libaacs-0.10.0.ebuild 996 BLAKE2B 0b67440a4d94a4cb6908ab9b977458de835fd9ea2bc161d00a10d9604c2a085c5b1bf3d6b8d4ec4470bf85f3cb5afc4bd586859635a0a3aa3fda319486e786ec SHA512 72b9bbca677307e8a58183b8cae32eda06af8d82586c95021bcc62f99bbf2d6f2bf22ba3c31c63239e4b6d82a152bb7bbc5469928ce06cc81fcc1e763e5af236
+EBUILD libaacs-0.11.0.ebuild 985 BLAKE2B 062f2ff64b2585005ce91d4d1ddedd9960ae2e7965a4071ad0ac862fb5ea58f44b3ad0ab844d874979084af3b4d9106a1b85f9655fc0eca5440ed127a7ae7f8a SHA512 69c5b3d2e4b43a27380540b5526d663100c861ea89e26ccaf7a57f6072f078c603acf4b12e1e2c91a6dc5a1a0f2e1e017413e4d7a5ed641dfcbb131645cc07cf
+EBUILD libaacs-0.9.0.ebuild 806 BLAKE2B 1b4852f3be27474d53b4d185defdf8f68802ca9feb0fcafcfdc1b5c39f0033a3fe13d68e71ddc4dd3ab5c6edd79f8270b1dac9588c18435225d0ab57d935eddc SHA512 f8760dbd524681a777f6ac97d003d0b35c0519e2af18231bd0285063950307f1c1583f55bcebd9a33ca400223335600d45ee35e12f39e0f4ea9b420ead6e32b7
+MISC metadata.xml 255 BLAKE2B 84855d12acedab301e8e709820ef74f658689209ce914cad2c71d11f4248a7d3356109d93573ed675698031eca1e0b6e0372a61c2a43a9f09840e295f94b74ef SHA512 92b56f4d9c86ad69341e67e8cc74db0bd068e45afdf2f3d3f9ea410fc5e64f3476143b656f2cb3d26b97ae01a967d71766952eba22e2278aab907c20b00ae0e9
```

Comment:
Manifests should only contain distfiles, we use so-called thin manifest, have a look at https://devmanual.gentoo.org/general-concepts/manifest/index.html

Gentoo's ebuild repository is automatically configured to use thin manifests (https://github.com/gentoo/gentoo/blob/master/metadata/layout.conf#L48), check if that is the changed in your local repo, and then regerenerate manifest

Reply:
Ok, I think I have it now, thanks for the tips and references. I was using rsync for portage but have now updated to using git and enabled thin manifests etc. Took me a bit to work out how to join the commits but that will definitly be helpful in the future.